### ISSUE TITLE
Core Dump on training step

### ISSUE THREAD

**Issue by jsilter** [deepfakes/faceswap-playground#207]
I've installed deepfakes onto an Ubuntu 16 system. I'm running using a python virtual environment (no docker) and without CUDA (note: I don't have the dependencies installed and my GPU sucks but if there is a specific flag I need to enable CPU training I didn't use it because I didn't see any mention in the docs). Using tensorflow 1.5 (higher tensorflow versions get errors).

I've been able to run the extract step, and the convert step with pre-packaged cage/trump models, but I haven't been able to run training myself. When I try I get:

QObject::moveToThread: Current thread (0x2fc5c80) is not the object's thread (0x2b805f0).
Cannot move to target thread (0x2fc5c80)

Segmentation fault (core dumped)

## Actual behavior

*Describe, in some detail, what the program does instead. Be sure to include any error message or screenshots.*

## Steps to reproduce
Download training data from https://anonfile.com/p7w3m0d5be/face-swap.zip
Extract faces:
python faceswap.py extract -i ./data/face-swap/data/cage -o ./data/out/cage

Try training
python faceswap.py train -A ./data/out/trump -B ./data/out/cage -m ./output/models -p

*Describe, in some detail, the steps you tried that resulted in the behavior described above.*

## Other relevant information
- **Operating system and version:** Ubuntu Linux 
- **Python version:**3.5,
- **Faceswap version:** commit 472a3ec9bcb841667a1549cdf6fab97fa2cfdfee
- **Faceswap method:** CPU (I think?)
- pip freeze output:
absl-py==0.4.1
astor==0.7.1
bleach==1.5.0
click==6.7
cloudpickle==0.5.5
cmake==3.12.0
cycler==0.10.0
dask==0.19.0
decorator==4.3.0
dlib==19.15.0
face-recognition==1.2.3
face-recognition-models==0.3.0
ffmpy==0.2.2
gast==0.2.0
grpcio==1.14.2
h5py==2.8.0
html5lib==0.9999999
Keras==2.2.0
Keras-Applications==1.0.4
Keras-Preprocessing==1.0.2
kiwisolver==1.0.1
Markdown==2.6.11
matplotlib==2.2.2
networkx==2.1
numpy==1.14.5
nvidia-ml-py3==7.352.0
opencv-python==3.4.1.15
pathlib==1.0.1
Pillow==5.2.0
pkg-resources==0.0.0
protobuf==3.6.1
pyparsing==2.2.0
python-dateutil==2.7.3
pytz==2018.5
PyWavelets==1.0.0
PyYAML==3.13
scandir==1.7
scikit-image==0.14.0
scipy==1.1.0
six==1.11.0
tensorboard==1.6.0
tensorflow==1.5.0
tensorflow-tensorboard==1.5.1
termcolor==1.1.0
toolz==0.9.0
tqdm==4.25.0
Werkzeug==0.14.1

Any thoughts?

**Comment by LeonNardella** [comment]
Could you look into your dmesg and see what library is crashing when the segfault happens?
I'd guess tensorflow is using some new instruction your CPU doesn't support.

**Comment by jsilter** [comment]
Well I just went back and re-ran it and the segfault is no longer happening. I double-pinky swear it was highly reproducible last week.
Now I have a new problem, which is that my computer simply shuts down after 15 minutes of training. Perhaps the segfault was a lesser of two evils and the CPU is overheating. Regardless, it doesn't seem like this machine is capable of running this program. I've been able to run it on AWS without issue.
